node/node_v2.go

```diff
@@ -115,27 +104,38 @@ func (n *Node) DownloadBundles(
 
 	}
 
-	// storeChunksSemaphore can be nil during unit tests, since there are a bunch of places where the Node struct
-	// is instantiated directly without using the constructor.
-	if n.storeChunksSemaphore != nil {
-		// So far, we've only downloaded metadata for the blob. Before downloading the actual chunks, make sure there
-		// is capacity in the store chunks buffer. This is an OOM safety measure.
+	return downloadSizeInBytes, relayRequests, nil
+}
 
-		probe.SetStage("acquire_buffer_capacity")
-		semaphoreCtx, cancel := context.WithTimeout(ctx, n.Config.StoreChunksBufferTimeout)
-		defer cancel()
-		err := n.storeChunksSemaphore.Acquire(semaphoreCtx, int64(downloadSizeInBytes))
-		if err != nil {
-			return nil, nil, fmt.Errorf("failed to acquire buffer capacity: %w", err)
+func (n *Node) DownloadChunksFromRelays(
```

Comment:
docs for `DownloadChunksFromRelays` and `DetermineChunkLocations` could use a little love, but the lack of docs is a preexisting condition, so up to you if you want to defer that

Reply:
expanded docs